File: CobongMemo/app/src/main/java/cobong/jeongwoojin/cobongmemo/cobongmemo/view/memo/MemoBinding.java
package cobong.jeongwoojin.cobongmemo.cobongmemo.view.memo;

import androidx.databinding.BindingAdapter;
import android.widget.ImageView;

import cobong.jeongwoojin.cobongmemo.cobongmemo.R;

public class MemoBinding {

    @BindingAdapter("memoType")
    public static void setMemoType(ImageView view, String memoType) {
        //check memoType
        if (memoType.equals("text")) {
            view.setImageDrawable(view.getResources().getDrawable(R.drawable.text));
        } else if (memoType.equals("handwrite")) {
            view.setImageDrawable(view.getResources().getDrawable(R.drawable.handwriting));
        } else {
            view.setImageDrawable(view.getResources().getDrawable(R.drawable.voice));
        }

    }

}

File: CobongMemo/app/src/main/java/cobong/jeongwoojin/cobongmemo/cobongmemo/view/memo/MemoBinding.kt
package cobong.jeongwoojin.cobongmemo.cobongmemo.view.memo

import android.widget.ImageView
import androidx.core.content.ContextCompat
import androidx.databinding.BindingAdapter
import cobong.jeongwoojin.cobongmemo.cobongmemo.R

object MemoBinding {

    @JvmStatic
    @BindingAdapter("memoType")
    fun setMemoType(view: ImageView, memoType: String) {
        //check memoType
        if (memoType == "text") {
            view.setImageDrawable(ContextCompat.getDrawable(view.context, R.drawable.ic_text_format_black_24dp))
        } else if (memoType == "handwrite") {
            view.setImageDrawable(ContextCompat.getDrawable(view.context, R.drawable.ic_palette_black_24dp))
        } else {
            view.setImageDrawable(ContextCompat.getDrawable(view.context, R.drawable.ic_keyboard_voice_black_24dp))
        }

    }

}
